src/main/cpp/alias/Alias.cpp

```diff
@@ -0,0 +1,82 @@
+/*
+ * Alias.cpp
+ *
+ *  Created on: Mar 11, 2017
+ *      Author: cedric
+ */
+
+#include "Alias.h"
+
+#include <exception>
+#include <math.h>
+#include <assert.h>
+#include <iostream>
+
+namespace stride{
+namespace alias{
+
+Alias::Alias(std::vector<double> probabilities) : Alias(probabilities,std::random_device()()) {}
```

Comment:
I think you need to `#include <random>` before you can use `std::random_device`.